spec/hydra/works/models/generic_work_spec.rb

```diff
@@ -13,15 +13,22 @@
 
   describe '#generic_works=' do
     it 'should aggregate generic_works' do
-      generic_work1.generic_works = [generic_work2, generic_work3]
-      expect(generic_work1.generic_works).to eq [generic_work2, generic_work3]
+      generic_work1.child_generic_works = [generic_work2, generic_work3]
+      expect(generic_work1.child_generic_works).to eq [generic_work2, generic_work3]
     end
   end
 
-  describe '#generic_files=' do
+  describe '#child_generic_files=' do
```

Comment:
I don't have a big preference for this particular method name, but we do have parent/child options for linking between works and generic_files.  So I think child_ was added to this method name to make those consistent:

```
generic_work#parent_generic_works
generic_work#child_generic_works
generic.work#child_generic_files

generic_file#parent_generic_works
generic_file#parent_generic_files
generic_file#child_generic_files
```

Happy to talk about this at standup today, and hear from @elrayle @terrellt and others.